### DISCUSSION TITLE
List of Requested Guides in Docs

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3585] [Feature Requests]
Until we have a better location, this can be where we post ideas for new "Guides" in the Docs. I'll kick it off with:

## Creating a content workflow

**Comment by DaanMoura** [comment]
- Migration guide (from older directus version)
- Better examples on extensions guide

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
